From e9a2039e86f9f153e55ef2182347196bd0db7dbd Mon Sep 17 00:00:00 2001
From: "Kevin F. Haggerty" <haggertk@lineageos.org>
Date: Wed, 3 Jan 2018 06:28:36 -0700
Subject: [PATCH 16/19] klte-common: Move charger service into the charger
 domain

* O split charger and healthd into different processes, with healthd
  running only in an full Android environment and charger running in
  charge-only and recovery. It also updated the healthd sepolicy and
  created a separate charger one.
* We've been using the charger symlink for ages, but the healthd
  domain is no longer appropriate.
* See for details:
  LineageOS/android_system_sepolicy@dc08245c
  LineageOS/android_system_core@526f8cf3

Change-Id: I6f528ffdcb7b14f9a51fd4246770ceecf8fb968b
---
 rootdir/etc/init.qcom.rc | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/rootdir/etc/init.qcom.rc b/rootdir/etc/init.qcom.rc
index fe3036f..e8d7b5e 100644
--- a/rootdir/etc/init.qcom.rc
+++ b/rootdir/etc/init.qcom.rc
@@ -1,5 +1,5 @@
 # Copyright (c) 2009-2012, Code Aurora Forum. All rights reserved.
-#               2017, The LineageOS Project. All rights reserved.
+#               2017-2018, The LineageOS Project. All rights reserved.
 #
 # Redistribution and use in source and binary forms, with or without
 # modification, are permitted provided that the following conditions are met:
@@ -1019,7 +1019,7 @@ on property:sys.boot_completed=1
 
 service charger /charger
     class charger
-    seclabel u:r:healthd:s0
+    seclabel u:r:charger:s0
 
 service sec-sh /system/bin/sh /system/etc/init.sec.boot.sh
     class main
-- 
2.7.4

